src/mpc-hc/CMPCThemeMenu.cpp

```diff
@@ -503,7 +521,8 @@ void CMPCThemeMenu::GetStrings(MenuObject* mo, CString& left, CString& right)
 
 void CMPCThemeMenu::MeasureItem(LPMEASUREITEMSTRUCT lpMeasureItemStruct)
 {
-    initDimensions();
+    initDimensions(); //should happen before drawitem

```

Comment:
Are these initDimensions calls redundant? Maybe use `assert(font.m_hObject)` instead?

Reply:
initDimensions gets called on first MeasureItem, and shouldn't need to be called again.  But out of paranoia I initialized it in any function that can use the font object (including drawitem).  I think it's impossible but I am not 100% sure.

I added an if block to be sure, now.